libs/EXTERNAL/libsdcparse/src/sdc_common.cpp

```diff
@@ -96,6 +96,20 @@ void add_sdc_create_clock(Callback& callback, const Lexer& lexer, CreateClock& s
      */
     callback.create_clock(sdc_create_clock);
 
+    //Save clock targets
+    auto targets = sdc_create_clock.targets.strings;
+    //Clean targets
+    sdc_create_clock.targets = StringGroup();
+    //Set 180 degrees phase shift and configure those clocks as virtual
+    sdc_create_clock.rise_edge = sdc_create_clock.fall_edge;
```

Comment:
Added comments describing the inversion. I'm not sure if renaming `rise_edge` and `fall_edge` from `sdcparse::CreateClock` is necessary because those precisely describe the waveform of the clock.
I also slightly modified the 180 degree phase shift.